2018 Robot/src/org/usfirst/frc/team4999/commands/intake/ToggleArms.java

```diff
@@ -0,0 +1,28 @@
+package org.usfirst.frc.team4999.commands.intake;
+
+import org.usfirst.frc.team4999.robot.Robot;
+import org.usfirst.frc.team4999.robot.subsystems.Claw;
+
+import edu.wpi.first.wpilibj.command.InstantCommand;
+
+/**
+ *
+ */
+public class ToggleArms extends InstantCommand {
+
+	private Claw claw = Robot.claw;
+
+    public ToggleArms() {
+        super();
+        requires(claw);
+    }
+
+    // Called once when the command executes
+    protected void initialize() {
+    	if(claw.isGripped())
```

Comment:
Toggle arms seems dangerous to me from a usability standpoint.  Too much room for out-of sync or unwanted toggling.  We should talk through what the expectations are from the driver standpoint in terms of how they'd want to communicate with the robot and then implement that.